_pages/icml_2020/icml-2020-sponsor-es.html

```diff
@@ -0,0 +1,11 @@
+---
+layout: main
+permalink: /icml-2020-sponsor-es
+title: LXAI @ ICML Patrocinio
+published: true
```

Comment:
Until information is updated on the Spanish version of this site, published should be false.